### ISSUE TITLE
Reusable classes

### ISSUE THREAD

**Issue by zodern** [Finger-Puppet-Mafia-FTC-Team/robot-controller#2]
There is some non trivial code that would be nice to have a reusable class for. This will be a running list of ideas.

**Comment by zodern** [comment]
Using a game pad button to toggle a function. If it is held down longer than 1 loop cycle, it will be toggled more than once, meaning it would appear to not have been toggled. The class should make sure there has been at least one loop cycle with the key up before toggling again.

**Comment by zodern** [comment]
Handle if a piece of hardware is not detected and send a message using telemetry to the driver station. This should be done before the instructions are sent as suggested in #3 .
